staubli_tx2-90_support/urdf/tx2-90l_macro.xacro

```diff
@@ -0,0 +1,239 @@
+      </collision>
+      <inertial>
+          <mass value="7.501506"/>
+          <origin xyz="0.000002 0.065008 -0.071428" rpy="0.0 0.0 0.0"/>
+          <inertia ixx="0.058902" ixy="0.000000" ixz="0.000001"
+                                  iyy="0.056884" iyz="-0.003672"
+                                                 izz="0.034271"/>
+      </inertial>
+    </link>
+    <link name="${prefix}link_4">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2-90_support/meshes/tx2-90l/visual/link_4.stl"/>
+        </geometry>
+        <xacro:material_staubli_ral_melon_yellow />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2-90_support/meshes/tx2-90l/collision/link_4.stl"/>
+        </geometry>
+      </collision>
+      <inertial>
+          <mass value="6.420429"/>
+          <origin xyz="-0.001998 0.049992 0.360301" rpy="0.0 0.0 0.0"/>
+          <inertia ixx="0.103999" ixy="0.000003" ixz="-0.001379"
+                                  iyy="0.098782" iyz="-0.000007"
+                                                 izz="0.019071"/>
+      </inertial>
+    </link>
+    <link name="${prefix}link_5">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2-90_support/meshes/tx2-90/visual/link_5.stl"/>
+        </geometry>
+        <xacro:material_staubli_ral_grey_aluminium />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2-90_support/meshes/tx2-90/collision/link_5.stl"/>
+        </geometry>
+      </collision>
+      <inertial>
+          <mass value="0.568760"/>
+          <origin xyz="0.000000 0.049931 0.022823" rpy="0.0 0.0 0.0"/>
+          <inertia ixx="0.001084" ixy="0.000000" ixz="0.000000"
+                                  iyy="0.001123" iyz="0.000001"
+                                                 izz="0.000383"/>
+      </inertial>
+    </link>
+    <link name="${prefix}link_6">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2-90_support/meshes/tx2-90/visual/link_6.stl"/>
+        </geometry>
+        <xacro:material_staubli_ral_grey_aluminium />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://staubli_tx2-90_support/meshes/tx2-90/collision/link_6.stl"/>
+        </geometry>
+      </collision>
+      <inertial>
+          <mass value="0.030977"/>
+          <origin xyz="-0.000243 0.050000 -0.206293" rpy="0.0 0.0 0.0"/>
+          <inertia ixx="0.000012" ixy="-0.000000" ixz="0.000000"
+                                  iyy="0.000011"  iyz="0.000000"
+                                                  izz="0.000022"/>
```

Comment:
These values are somewhat small. Have you tried this in Gazebo? I've seen reports of too small inertias causing instabilities.